src/extensions/surveys.tsx

```diff
@@ -817,32 +820,32 @@ export function usePopupVisibility(
             localStorage.setItem('lastSeenSurveyDate', new Date().toISOString())
             setTimeout(() => {
                 const inputField = document
-                    .querySelector(getPosthogWidgetClass(survey.id))
+                    .querySelector(`.${getSurveyContainerClass(survey)}`)
```

Comment:
should `getSurveyContainerClass` already start with the `.` so you don't have to concatenate in multiple places? it was like this before as well

Reply:
yeah there's two places where it requires the . vs one that doesn't, so will change

Reply:
i changed instead so the function has the option to return the [class as a selector](https://github.com/PostHog/posthog-js/pull/1924/commits/e1f6dca0de4a4ec8734e7c0b4db6a85756a9be6f)